src/Data/Unrestricted/Linear/Internal/Instances.hs

```diff
@@ -186,7 +186,8 @@ instance (KnownNat n, Consumable a) => Consumable (V n a) where
 
 instance (KnownNat n, Dupable a) => Dupable (V n a) where
   dupR (V xs) =
-    V . Unsafe.toLinear (Vector.fromListN (V.theLength @n))
+    V
+      . Unsafe.toLinear (Vector.fromListN (V.theLength @n))
```

Comment:
What happened here?

Reply:
Our ormolu version is given by stackage AFAIK, which I bumped.
I suppose the new ormolu version has some breaking changes related to default operator fixity/formatting.
(I didn't change that manually)